doc/user-guide/push-notifications/Push-notifications.md

```diff
@@ -57,16 +57,16 @@ node. Check out [this][WithMongoosePush] tutorial on how to setup [MongoosePush]
   backend.push.plugin_module = "mod_event_pusher_push_plugin_defaults" # optional
```

Comment:
Above this line, there's a `  plugins = ["push] # mandatory minimal config` line where `"push` is missing the closing quotation mark and making the rendering of the snippet ugly, can you fix? 🙂

Reply:
what a brilliant catch 🙂 will fix while revisiting these pages